docs/server/settings.rst

```diff
@@ -250,6 +250,164 @@ Configuration settings for applications used by Pootle.
+
+  Absolute path to the `PhantomJS binary <http://phantomjs.org/>`_. This is
+  needed in order to optionally generate PDF invoices out of user activity.
+
+
+.. setting:: POOTLE_REPORTS_INVOICES_DIRECTORY
+
+``POOTLE_REPORTS_INVOICES_DIRECTORY``
+  Default: ``working_path('invoices')``
+
+  .. versionadded:: 2.7.3
+
+  Base directory where the invoices will be created. Invoices will be generated
+  in subdirectories for each year and month combination, i.e.
+  ``$POOTLE_REPORTS_INVOICES_DIRECTORY/<YYYY-MM>/``.
+
+
+.. setting:: POOTLE_REPORTS_INVOICES_COMPANY
+
+``POOTLE_REPORTS_INVOICES_COMPANY``
+  Default: ``'ACME'``
+
+  .. versionadded:: 2.7.3
+
+  Name of your company as displayed in the invoice. Check the templates at
+  *templates/invoices/* for further details and customizations.
+
+
+.. setting:: POOTLE_REPORTS_INVOICES_DEPARTMENT
+
+``POOTLE_REPORTS_INVOICES_DEPARTMENT``
+  Default: ``'ACME Department'``
+
+  .. versionadded:: 2.7.3
+
+  Name of your company deparment as displayed in the invoices. Check the
+  templates at *templates/invoices/* for further details and customizations.
+
+
+.. setting:: POOTLE_REPORTS_INVOICES_RECIPIENTS
+
+``POOTLE_REPORTS_INVOICES_RECIPIENTS``
+  Default: ``{}`` (empty dictionary)
+
+  .. versionadded:: 2.7.3
+
+  The list of usernames for whom invoices will be generated and send to.
+
+  The setting holds a dictionary where the keys are actual usernames of the
+  running Pootle instance and the values are dictionaries of key-value pairs
+  which will hold the user-specific configuration used to construct individual
+  invoices.
+
+  The user configuration accepts the following keys:
+
+  ``name`` (required)
+    Full name as displayed in the invoice. The official "full" person name can
+    be different from the public *shortened* version available in the Pootle
+    profile, hence this name is used instead as it's dealing with financial
+    documents.
+
+  ``wire_info`` (required)
+    Wire information which includes payment details. This can be a multi-line
+    string and it will be rendered as such in the invoice, with leading
+    whitespace removed.
+
+  ``paid_by`` (required)
+    Official name of the company/person paying the invoice.
+
+  ``invoice_prefix``
+    Prefix used to generate invoice IDs. This will be prepended to the current
+    month in `YYYY-MM` format as a string.
+
+    A recommended approach for prefixing invoices is the *"Language-Initials"*
+    format. For example, if there's a translator named *"John Doe"* translating
+    into Spanish (ES), the prefix might look like ``'ES-JD-'`` and the invoices
+    generated for this person will be consequently named like *'ES-JD-2016-06'*,
+    *'ES-JD-2016-07'* and so on. If there is a need for these IDs to be globally
+    unique, one needs to specify a unique prefix for each of the paid
+    translators.
+
+  ``minimal_payment``
+    Minimal payment threshold value to be reached for the payment to be sent out
```

Comment:
@julen Let's change "sent out" to "initiated" to avoid the potential confusion you mentioned in the previous review cycle.